code/modules/mob/living/silicon/robot/robot_defense.dm

```diff
@@ -48,10 +48,29 @@
 			module?.update_cells(TRUE)
 			diag_hud_set_borgcell()
 
-	if(!opened)
-		if(..()) // hulk attack
-			spark_system.start()
-			spawn(0)
-				step_away(src, user, 15)
-				sleep(3)
-				step_away(src, user, 15)
+	if(..()) // hulk attack
+		spark_system.start()
+		spawn(0)
+			step_away(src, user, 15)
+			sleep(3)
+			step_away(src, user, 15)
+
+/mob/living/silicon/robot/run_resist()
+	..()
+	if(HAS_TRAIT(src, TRAIT_BORG_TIPPED))
+		untilt(src, 20 SECONDS)
```

Comment:
Oh boy, almost half a minute...

Reply:
Please keep reviews for code related complaints, if you want to suggest better options to the Author, you can always copypaste the code into codeblocks